components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1310,116 +1315,137 @@ END SUBROUTINE p3_main
 
   SUBROUTINE access_lookup_table(dumjj,dumii,dumi,index,dum1,dum4,dum5,proc)
 
+#ifdef SCREAM_CONFIG_IS_CMAKE
+    use micro_p3_iso_f, only: access_lookup_table_f
+#endif
+
     implicit none
 
     real(rtype)    :: dum1,dum4,dum5,proc,iproc1,gproc1,tmp1,tmp2
     integer :: dumjj,dumii,dumi,index
 
-    ! get value at current density index
+#ifdef SCREAM_CONFIG_IS_CMAKE
```

Comment:
I'm confused why you have both the ifdef and the "use_cxx" conditionals here. It seems to me that we either have the intention to use cxx or not, so we should only need 1 conditional...

Reply:
@PeterCaldwell , use_cxx is a runtime setting, so the compiler will expect functions like access_lookup_table_f to be available to the module, even if use_cxx is false. If we are compiling micro_p3 in the context of cam, not scream, the micro_p3_iso_f module will not be available to this module (micro_p3_iso_f is part of scream), so that's the reason for the ifdef.